applications/charter/P4_Apps_WG_charter.mdk

```diff
@@ -15,18 +15,28 @@ P4 Applications Working Group
 
 - To subscribe, visit
   [http://lists.p4.org/mailman/listinfo/p4-apps_lists.p4.org](http://lists.p4.org/mailman/listinfo/p4-apps_lists.p4.org)
-- To post a message to all list members, end an email to <p4-apps@lists.p4.org>
+- To post a message to all list members, send an email to <p4-apps@lists.p4.org>
 
 # Description of the Working Group
 The P4 development environment is maturing. Developers have started to create
 new, innovative data-plane applications using P4. These applications will 
-change the way people design, build, and run not just their networks, but 
-their compute and storage infrastructure as well. In addition, these 
-applications will be key drivers for future enhancements to the language. 
-We have already seen some great contributions in this application area and 
-are eager to see the community contribute more such examples in the future.
-Meanwhile some applications inherently work across multiple devices, layers,
-and domain boundaries. In such cases, it’ll be helpful to have a forum where
+change the way people design, build, and run their networks, as well as
+compute and storage infrastructure. In addition, these applications will
+be key drivers for future enhancements to the P4 language, architecture and API.
+We have already seen some great contributions in this application area, for
```

Comment:
We have already seen some great contributions in the area of P4 applications such as: